x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/SecurityImplicitBehaviorBootstrapCheckTests.java

```diff
@@ -26,7 +26,7 @@
 public class SecurityImplicitBehaviorBootstrapCheckTests extends AbstractBootstrapCheckTestCase {
 
     public void testFailureUpgradeFrom7xWithImplicitSecuritySettings() throws Exception {
-        final Version previousVersion = VersionUtils.randomVersionBetween(random(), Version.V_7_2_0, Version.V_7_16_0);
+        final Version previousVersion = Version.CURRENT.minimumCompatibilityVersion();
```

Comment:
Would be good to keep the extra generality in the choice of version here rather than fixing it to be exactly 7.17.0:

```suggestion
        final Version previousVersion = randomValueOtherThan(
            Version.V_8_0_0,
            () -> VersionUtils.randomVersionBetween(random(), Version.CURRENT.minimumCompatibilityVersion(), Version.V_8_0_0)
        );
```